DocSum/tests/test_compose_on_amd_rocm.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
```

Comment:
To align the naming format, please name the script as `DocSum/tests/test_compose_on_rocm.sh`

Reply:
test_compose_on_amd_rocm.sh renamed to test_compose_on_rocm.sh